### PR TITLE
Home Assistant integration via MQTT

### PR DESCRIPTION
**Pull request by JSa1987**
Added a Home Assistant integration via MQTT. The advantage over using APIs for integration is that this way the configuration needed on the HA side is minimal. PiHome will automatically broadcast over MQTT all the sensor definition needed to setup and then update the following HA sensors:
- PiHome CPU Usage
- PiHome CPU Load (1m, 5m and 15m)
- PiHome CPU temperature
- PiHome Memory Use
- PiHome Swap Usage
- PiHome Disk Use
- PiHome Host Ip
- PiHome Last Boot
- PiHome Network throughput (up & down)
- PiHome Wifi Strength
- Boiler status
- Zone Temperature (for each zone)
- Zone status (for each zone)
- Zone sensor battery percentage (for each zone using a MySensor sensor)
- Zone sensor battery voltage (for each zone using a MySensor sensor)

**Changes**
- `cron/home-assistant_sensors.py` script was created that handles all the MQTT communication. This is based on the [**system_sensors from Sennevds** ](https://github.com/Sennevds/system_sensors).
- `cron/check_HA.php` script was addd to check that `cron/home-assistant_sensors.py` is running and if not re-start it.
- `ajax.php` was modified to add the option to define the MQTT broker used by for HA.
- `cron/CronJobs List` was modified to run  `cron/check_HA.php` every minute
- `setup.php` was modified to add the option for `cron/check_HA.php` to crontab and install the Python modules needed (defined in `requirements.txt`). Not sure if there is a better way to install the Python modules required during the setup, I'm open to ideas here.

### CODE REVIEW

**Comment by pihome-shc** [comment]
@JSa1987 can we implement on MaxAir version?

**Comment by JSa1987** [comment]
@pihome-shc I would think this should be possible. My understanding is that the database structure has change so I would need to adjust a bit how data is pulled. Can you confirm that `pihome-shc / PiHomeHVAC` is the current repository for AirMax?

Unfortunately I don't have an installation of MaxAir to experiment and test. What is the status of development for AirMax, is this mature enough to be used to control the heat system? If so, I could consider upgrading so that I can then get a better understanding of how the new system works and then port this over.

**Comment by pihome-shc** [comment]
Yes, MaxAir is mature now and using it in production. Yes pihome-shc/PiHomeHVAC is correct repository. 
Thank you for working on this.

**Comment by JSa1987** [comment]
Very good then. I will look into upgrading to MaxAir and start porting this also there. 
I will close this pull request for now. Once I will had a look at MaxAir I will revisit also the implementation for PiHome so that they are kept as similar as possible and I will then re-submit another pull request.

**Comment by pihome-shc** [comment]
we can still update pihome as work is already done so why not.
